src/consumer/component/dependencies/files-dependency-builder/detector-hook.ts

```diff
@@ -43,7 +49,7 @@ export interface DependencyDetector {
 export class DetectorHook {
   static hooks: DependencyDetector[] = [];
 
-  isSupported(ext: string): boolean {
+  isSupported(ext: string): boolean | string {
```

Comment:
what string can be returned from here?
maybe in that case we should rename the function from `is` to somthing else?

Reply:
Sorry, this was a mistake when I tried some other ideas in the early stage. I already reverted it.